lib/screens/UsersProfileScreen.dart

```diff
@@ -0,0 +1,1576 @@
+import 'package:flutter/material.dart';
+import 'dart:convert';
+import 'package:smart_chef/utils/APIutils.dart';
+import 'package:smart_chef/utils/colors.dart';
+import 'package:smart_chef/utils/userAPI.dart';
+import 'package:smart_chef/utils/authAPI.dart';
+import 'package:http/http.dart' as http;
```

Comment:
Please read the section in the style guide about [import order](https://dart.dev/guides/language/effective-dart/style#ordering) and fix your imports here and elsewhere accordingly